tools/validators/instance_validator/tests/fake_instances/BAD/bad_abstract_type.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ZONE is an abstract type.
+ZONE-105:
+  type: HVAC/ZONE
+  id: CDM/f7ed1725-53f9-4978-98c7-648f5d66e98e
```

Comment:
probably good to use something fake here...

Reply:
Fakes are already tested, they are not found in the universe